nnunet/training/network_training/nnUNetTrainer.py

```diff
@@ -296,8 +296,7 @@ def plot_network_architecture(self):
             if torch.cuda.is_available():
                 torch.cuda.empty_cache()
 
-    def save_debug_information(self):
```

Comment:
Please revert the change where you integrate `save_debug_information` into `run_training`. We need it as a separate function because we can only execute this once per node when running DDP

Reply:
All changes regarding `save_debug_information()` are now reverted (I think it was a failed merge, since I not actively changed this)...